doc/releases/release-notes-3.2.rst

```diff
@@ -406,6 +406,18 @@ Boards & SoC Support
   * MAX7219 LED display driver shield
   * Panasonic Grid-EYE (AMG88xx)
 
+Build system and infrastructure
+*******************************
```

Comment:
should this be merged with the ["Build System" section](https://github.com/zephyrproject-rtos/zephyr/blob/bb5922db66a80d9d58e325b27de073d6c062e6ed/doc/releases/release-notes-3.2.rst#build-system) below?

Reply:
Yes, missed that. Thanks, fixed!